engine/table/src/main/java/io/deephaven/engine/table/impl/BlinkTableTools.java

```diff
@@ -26,150 +24,86 @@
  * @see Table#BLINK_TABLE_ATTRIBUTE
  */
 public class BlinkTableTools {
+    public static final Object DEFAULT_MEMO_KEY = new Object() {
+        @Override
+        public String toString() {
+            return "DEFAULT_MEMOIZATION_KEY";
+        }
+    };
```

Comment:
Could just be
```
new String("DEFAULT_MEMOIZATION_KEY");
```

Reply:
I'm a fan of how it's setup here w/ Object; even though unlikely, a string-based memo-key is "public" and the user can just use the same string.

Reply:
I think that's an argument for using object identity when comparing memo keys. Also, why is it bad if the user really wants to explicitly use our default memo key? It's public here, for some reason, although I'd think it could be private.

Reply:
Oh, I see it's explicitly public. 

I don't think we'd want to use object identity for all memo, otherwise it wouldn't be obvious how one achieves memo over API today.

Reply:
OK. I'm convinced enough. We stick to Object equality, keep this default as an Object so it will rely on reference equality, and maybe make it private (but I'm ambivalent about that).